indico/modules/events/util.py

```diff
@@ -83,6 +84,8 @@ def get_object_from_args(args=None):
         obj = event
     elif object_type == 'session':
         obj = Session.query.with_parent(event).filter_by(id=args['session_id']).first()
+    elif object_type == 'block':
```

Comment:
maybe `session_block` would be better? Also, I'd nest the URL for session blocks inside the session like we do with subcontributions. ie `/sessions/<int:session_id>/blocks/<int:block_id>` in the blueprint